packages/core/strapi/lib/types/core/attributes/base.d.ts

```diff
@@ -29,7 +29,7 @@ export type AttributeType =
 /**
  * Most basic shape of a schema attribute
  */
-export interface Attribute<T extends AttributeType = AttributeType> {
+export interface Attribute<T extends AttributeType = AttributeType, U extends {} = {}> {
```

Comment:
that's U used for ?

Reply:
I think it was an experiment to pass different generics to the attribute so that we can define additional properties. But it's no longer used/needed. I'll remove it.